Formula/gn.rb

```diff
@@ -0,0 +1,44 @@
+class Gn < Formula
+  desc "Generate Ninja - Chromium's build system"
+  homepage "https://gn.googlesource.com/gn/"
+  url "https://gn.googlesource.com/gn.git",
+    :revision => "f5fc06000379fe2c11fc130431e826e860ea6aae"
+  version "1547"
+  sha256 "8094c4ac075f918fdb28af10e16c9d4e41481702bd14a2c06cffd50fe03de9ba"
+  depends_on "ninja" => :build
+
+  def install
+    system "python", "build/gen.py"
+    system "ninja", "-C", "out/", "gn"
+    mkdir bin
+    cp "out/gn", bin/"gn"
```

Comment:
nit: I think these two lines are equivalent to `bin.install "out/gn"`

Reply:
They are, @ryland could you edit this?

Reply:
Done